src/keys/navigation-key.js

```diff
@@ -3,6 +3,12 @@ import {fireEvent} from '@testing-library/dom'
 import {setSelectionRangeIfNecessary} from '../utils'
 
 const keys = {
+  Home: {
+    keyCode: 35,
+  },
+  End: {
+    keyCode: 36,
+  },
```

Comment:
Following [keycode.info](https://keycode.info/)'s table, those keys are inverted is it right?
![image](https://user-images.githubusercontent.com/19766035/104028487-894f5c80-51a7-11eb-986d-adc36b81f082.png)

Reply:
@luistak I've corrected the blunder.